src/components/CIPipelineN/CustomInputOutputVariables.tsx

```diff
@@ -0,0 +1,322 @@
+                                                            ...base,
+                                                            color: 'var(--N900)',
+                                                            background: 'var(--N100) !important',
+                                                            padding: '0px 10px',
+                                                            display: 'flex',
+                                                            height: '32px',
+                                                            fontSize: '12px',
+                                                        }),
+                                                    }}
+                                                />
+                                            )}
+                                        </div>
+                                    </div>
+                                )}
+                                <input
+                                    style={{ width: '80% !important' }}
+                                    className="w-100 bcn-1 en-2 bw-1 pl-10 pr-10 pt-6 pb-6 bottom-radius-4 h-32"
+                                    autoComplete="off"
+                                    placeholder="Description"
+                                    type="text"
+                                    value={variable.description}
+                                    name="description"
+                                    onChange={(e) => handleInputOutputValueChange(e, index)}
+                                    onBlur={(e) => handleBlur()}
+                                />
+                            </div>
+
+                            <Close
+                                className="icon-dim-24 pointer mt-6 ml-6"
+                                onClick={() => {
+                                    deleteInputOutputValue(index)
+                                }}
+                            />
+                        </div>
+                        {formDataErrorObj[activeStageName].steps[selectedTaskIndex]?.inlineStepDetail[
+                            VariableFieldType[type]
+                        ][index] &&
+                            !formDataErrorObj[activeStageName].steps[selectedTaskIndex]?.inlineStepDetail[
+                                VariableFieldType[type]
+                            ][index].isValid && (
+                                <div className="pl-220 mb-20">
+                                    <span className="flexbox cr-5 mb-4 mt-4 fw-5 fs-11 flexbox">
+                                        <AlertTriangle className="icon-dim-14 mr-5 ml-5 mt-2" />
+                                        <span>
+                                            {
+                                                formDataErrorObj[activeStageName].steps[selectedTaskIndex]
+                                                    ?.inlineStepDetail[VariableFieldType[type]][index].message
```

Comment:
same here - move this to some var `formDataErrorObj[activeStageName].steps[selectedTaskIndex]?.inlineStepDetail[
                            VariableFieldType[type]
                        ][index]`